### ISSUE TITLE
[Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#33344]
#### VFS team name
Decision Tools

#### VFS product name
Find a VA Form PDF Download Instructions

#### Point of Contact/Reviewers
Angela Fowler (@AngelaFowler82) - Accessibility

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
While the download links state the form number and indicate their purpose, they don't indicate the name of the form. This could be confusing for those of us who are words not numbers people.

### VA.gov Experience Standard
[Category Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
If time and resources exist, we recommend to use the heading (which contains the form name) to describe the action link and download link by utilizing `aria-describedby`. This will help screen reader users with cognitive disabilities to confidently make selections. 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by MarciMcGuireGCIO** [comment]
This issue has been raised in previous enhancement reviews.  The root of the problem is that the VA controls the data that contains the form names, some of which have names like the ones listed below. Given their length, they don't lend themselves to being displayed in a link or button easily.  I will mark this blocked as it cannot be addressed until the VA makes a decision to shorten the names of these forms.

- Education Program Completion Notice/Service Obligation Placement—Health Professional Scholarship Program (HPSP) & Visual Impairment and Orientation and Mobility Professionals Scholarship Program (VIOMPSP)
- Dental Education Affiliation Agreement Between Department of Veterans Affairs (VA) and Institutions Sponsoring Dental Education
- Associated Health Education Affiliation Agreement Between the Department of Veterans Affairs (VA) and an Educational Program
